pkg/scripts/install.go

```diff
@@ -91,10 +92,18 @@ func (i *installer) run(scripts []string, dstDir string) []api.InstallResult {
 		if !i.fs.Exists(file) {
 			sourceResults[script].err = errors.NewDownloadError(file, -1)
 		}
+		file = filepath.Join(dstDir, api.DeprecatedSourceScripts, script)
+		if i.fs.Exists(file) {
+			glog.Infof("DEPRECATED: The .sti/bin directory will be deprecated, use .s2i/bin instead")
+			sourceResults[script] = &downloadResult{location: api.DeprecatedSourceScripts}
+		}
```

Comment:
need to refactor this anyway, so consider this pr as wip :)
On Jan 25, 2016 4:07 PM, "Ben Parees" <notifications@github.com> wrote:

> In pkg/scripts/install.go
> <https://github.com/openshift/source-to-image/pull/396#discussion_r50704081>
> :
>
> > @@ -91,10 +92,18 @@ func (i *installer) run(scripts []string, dstDir string) []api.InstallResult {
> >  		if !i.fs.Exists(file) {
> >  			sourceResults[script].err = errors.NewDownloadError(file, -1)
> >  		}
> > +		file = filepath.Join(dstDir, api.DeprecatedSourceScripts, script)
> > +		if i.fs.Exists(file) {
> > +			glog.Infof("DEPRECATED: The .sti/bin directory will be deprecated, use .s2i/bin instead")
> > +			sourceResults[script] = &downloadResult{location: api.DeprecatedSourceScripts}
> > +		}
>
> you should put this first so if they have files in both .s2i and .sti,
> they get a warning but the files in .s2i get used.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/source-to-image/pull/396/files#r50704081>.
>